mirdata/datasets/compmusic_carnatic_rhythm.py

```diff
@@ -0,0 +1,382 @@
+    def beats(self):
+        return load_beats(self.beats_path)
+
+    @core.cached_property
+    def meter(self):
+        return load_meter(self.meter_path)
+
+    @core.cached_property
+    def mbid(self):
+        return self._track_metadata.get("mbid")
+
+    @core.cached_property
+    def name(self):
+        return self._track_metadata.get("name")
+
+    @core.cached_property
+    def artist(self):
+        return self._track_metadata.get("artist")
+
+    @core.cached_property
+    def release(self):
+        return self._track_metadata.get("release")
+
+    @core.cached_property
+    def lead_instrument_code(self):
+        return self._track_metadata.get("lead_instrument_code")
+
+    @core.cached_property
+    def taala(self):
+        return self._track_metadata.get("taala")
+
+    @core.cached_property
+    def raaga(self):
+        return self._track_metadata.get("raaga")
+
+    @core.cached_property
+    def num_of_beats(self):
+        return self._track_metadata.get("num_of_beats")
+
+    @core.cached_property
+    def num_of_samas(self):
+        return self._track_metadata.get("num_of_samas")
+
+    @property
+    def audio(self):
+        """The track's audio
+
+        Returns:
+           * np.ndarray - audio signal
+           * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            audio_path=self.audio_path,
+            beat_data=[(self.beats, "beats")],
+            metadata={
+                "meter": self.meter,
+                "mbid": self.mbid,
+                "name": self.name,
+                "artist": self.artist,
+                "release": self.release,
+                "lead_instrument_code": self.lead_instrument_code,
+                "taala": self.taala,
+                "raaga": self.raaga,
+                "num_of_beats": self.num_of_beats,
+                "num_of_samas": self.num_of_samas,
+            },
+        )
+
+
+# no decorator here because of https://github.com/librosa/librosa/issues/1267
+def load_audio(audio_path):
+    """Load a Saraga Carnatic audio file.
+
+    Args:
+        audio_path (str): path to audio file
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+
+    """
+    if audio_path is None:
+        return None
+    return librosa.load(audio_path, sr=44100, mono=False)
+
+
+@io.coerce_to_string_io
+def load_beats(fhandle):
+    """Load beats
+
+    Args:
+        fhandle (str or file-like): Local path where the beats annotation is stored.
+
+    Returns:
+        BeatData: beat annotations
+
+    """
+    beat_times = []
+    beat_positions = []
+
+    reader = csv.reader(fhandle, delimiter=",")
+    for line in reader:
+        beat_times.append(float(line[0]))
```

Comment:
doesn't this data have the sam marked? then it should be like beats and downbeats? if no sam is marked then you can have it as beats